README.md

```diff
@@ -66,34 +66,40 @@ Install the ``python3-routes`` package:
 
 These instructions are meant for developers who are compiling the code for
 development and testing. To build binaries that are suitable for installation
-we recommend that you build .deb or .rpm packages, or refer to ``ceph.spec.in``
-or ``debian/rules`` to see which configuration options are specified for
-production builds.
+we recommend that you build `.deb` or `.rpm` packages, or refer to
+``ceph.spec.in`` or ``debian/rules`` to see which configuration options are
+specified for production builds.
 
-Build instructions:
+To build ceph, make sure that you are in the top-level `/ceph` directory, and
```

Comment:
To build Ceph

In the top-level `ceph` directory that contains `do_cmake.sh` and `CONTRIBUTING.rst`, then

Reply:
Yeah, that is better.

Accepted.